apps/condo/domains/ticket/schema/Ticket.test.js

```diff
@@ -1664,6 +1664,71 @@ describe('Ticket', () => {
+                const details = 'I have some problems with hot water!'
+
+                // User, should be banned from creating tickets to organization if he exceeds the limits.
+                const userClient = await makeClientWithResidentAccessAndProperty()
+                const unitName = faker.random.alphaNumeric(5)
+                await createTestResident(admin, userClient.user, userClient.property, {
+                    unitName,
+                })
+                for (let i = 0; i < DAILY_SAME_TICKET_LIMIT; ++i) {
+                    await createTestTicket(userClient, userClient.organization, userClient.property, { unitName, details })
+                }
+
+                await expectToThrowGQLError(async () => {
+                    await createTestTicket(userClient, userClient.organization, userClient.property, { unitName, details })
+                }, ERRORS.SAME_TICKET_FOR_PHONE_DAY_LIMIT_REACHED)
+
+                // User, banned from creating tickets to organization 1, can still create tickets to organization 2
+                const client2 = await makeClientWithProperty()
+                const unitName2 = faker.random.alphaNumeric(5)
+                await createTestResident(admin, userClient.user, client2.property, {
+                    unitName: unitName2,
+                })
+
+                const [secondOrgTicket] = await createTestTicket(userClient, client2.organization, client2.property, { unitName: unitName2, details })
+
+                expect(secondOrgTicket).toBeDefined()
+                expect(secondOrgTicket.id).toBeDefined()
+                expect(secondOrgTicket.details).toEqual(details)
+                expect(secondOrgTicket.organization.id).toEqual(client2.organization.id)
+                expect(secondOrgTicket.property.id).toEqual(client2.property.id)
+            })
+
+            test('user: resident should not be able to create tickets in single day over the limit', async () => {
+                // User, should be banned from creating tickets to organization if he exceeds the limits.
+                const userClient = await makeClientWithResidentAccessAndProperty()
+                const unitName = faker.random.alphaNumeric(5)
+                await createTestResident(admin, userClient.user, userClient.property, {
+                    unitName,
+                })
+                for (let i = 0; i < DAILY_TICKET_LIMIT; ++i) {
+                    await createTestTicket(userClient, userClient.organization, userClient.property, { unitName })
+                }
+
+                await expectToThrowGQLError(async () => {
+                    await createTestTicket(userClient, userClient.organization, userClient.property, { unitName })
+                }, ERRORS.TICKET_FOR_PHONE_DAY_LIMIT_REACHED)
+
+                // User, banned from creating tickets to organization 1, can still create tickets to organization 2
```

Comment:
It seems non-obvious to test two different cases in one "it". I have not seen anything like this in our code.

Reply:
There are some code like this in billing domain.. It comes handy and more efficient in terms of test execution time. 

But I agree, that this is kinda against best practices.

Reply:
IMO this scenario is ok. Since it will duplicate most of its logic (create N+1 ticket), receive a ban, then try another org.

Reply:
I aggree with @SavelevMatthew it's it looks like a real QA test case.